Library/Homebrew/sorbet/rbi/dsl/homebrew/dev_cmd/prof.rbi

```diff
@@ -0,0 +1,10 @@
+# typed: true
+
+# DO NOT EDIT MANUALLY
+# This is an autogenerated file for dynamic methods in `Homebrew::DevCmd::Prof`.
+# Please instead update this file by running `bin/tapioca dsl Homebrew::DevCmd::Prof`.
+
+class Homebrew::CLI::Args
+  sig { returns(T::Boolean) }
+  def stackprof?; end
```

Comment:
Cheating a bit here – command arg methods will be isolated to individual files per command, though (for now) arg methods all live in the same namespace (`Homebrew::CLI::Args`).

Reply:
@dduugg Potentially outrageous suggestion that can wait to another PR (including discussion can wait) to not block this one:

What if all the per-command arguments were subclasses of `Homebrew::CLI::Args`?

Reply:
That's the eventual idea, hence the "for now" above. 😉